_javascripts/page-loader.js

```diff
@@ -57,7 +67,10 @@ function versionSelector(list) {
       if(jqXHR.status == 404) {
         list.value = currentVersion;
         if(confirm("This page doesn't exist in version " + newVersion + ". Click OK to search the " + newVersion + " docs OR Cancel to stay on this page.")) {
-          window.location = "https://google.com/search?q=site:https://docs.openshift.com/container-platform/" + newVersion + " " + document.title.split(" | ")[0];
+          if (['3.65', '3.66', '3.67', '3.68', '3.69', '3.70', '3.71'].contains(newVersion)) {
+          window.location = "https://google.com/search?q=site:https://docs.openshift.com/acs/" + newVersion + " " + document.title;} else {
+            window.location = "https://google.com/search?q=site:https://docs.openshift.com/enterprise/" + newVersion + " " + document.title;
```

Comment:
@adellape This line 72 is what we have in the `main` branch, however, we have a different line in `enterprise-4.1` branch at https://github.com/openshift/openshift-docs/blob/enterprise-4.1/_javascripts/page-loader.js#L60

I am leaning towards keeping L60 from `enterprise-4.1` branch and then also cherry-picking these changes to the main branch. Can you look at this and suggest?

Reply:
Considering https://github.com/openshift/openshift-docs/pull/39658, I think that makes sense.